linux_os/guide/system/accounts/accounts-restrictions/password_storage/no_empty_passwords/ansible/sle15.yml

```diff
@@ -0,0 +1,17 @@
+# platform = multi_platform_sle
```

Comment:
Really wish ssg.build_remediations.get_rule_dir_remediations() support the concept of product family so we don't have to copy these around.

Reply:
Me too. We could use links, but I never really liked using those in git.

Reply:
Feel free to create a new issue asking for this. It seems reasonable.